web/web/api/v1/sns_read.py

```diff
@@ -0,0 +1,30 @@
+from flask import Flask, request, Blueprint
+import json
+
+sns_read_api_bp =  Blueprint('sns_read_api_bp', __name__)
```

Comment:
why `read` in the name? Let's just make it `sns_api_bp`

Reply:
done